src/test/java/com/openshift/internal/restclient/capability/resources/BuildCapabilitiesIntegrationTest.java

```diff
@@ -57,14 +57,14 @@ public void setUp() throws Exception {
         // a buildconfig
         IBuildConfigBuilder builder = client.adapt(IBuildConfigBuilder.class);
         assertNotNull("Exp. the client to be able to use a buildconfigbuilder", builder);
-        LOG.debug("Creating BuildConfig {}", bc);
+        LOG.debug("Creating BUILDCONFIG {}", bc);
```

Comment:
What's the rationale for that ?

Reply:
None, that's an unintentional change. Thanks for spotting. I'm correcting it.